activerecord/test/schema/schema.rb

```diff
@@ -288,6 +291,50 @@
     t.string :name
   end
 
+  create_table :destroy_later_parents, force: true, id: false do |t|
```

Comment:
I'd love to delete those new tables and reuse existing tables but I need to understand why those models are needed now.

Reply:
I added new models because adding the destroy later arguments to has many relations on other models was causing issues with other tests and I could not figure out a way to add it cleanly for just the tests it affects. It also felt much cleaner to isolate the tests.

Reply:
Also there were a lot of edge cases to test against 😅